### PR TITLE
Add Angular and Docker files, update ports and labels on aqueduct api deployment and service

### PR DESCRIPTION
**Pull request by MarcusSmith**

### CODE REVIEW

File: aqueduct/kubernetes/api-deployment-and-service.yaml [hunk 1]
```diff
@@ -6,10 +6,11 @@ metadata:
 spec:
   selector:
     app: <APP_NAME>
-    tier: frontend
+    tier: backend
```
Comment (joeconwaystk) [inline aqueduct/kubernetes/api-deployment-and-service.yaml]: change to role: backend to match angular deployment?

File: aqueduct/kubernetes/api-deployment-and-service.yaml [hunk 2]
```diff
@@ -22,7 +23,8 @@ spec:
     metadata:
       labels:
         app: <APP_NAME>
-        tier: frontend
+        tier: backend
```
Comment (joeconwaystk) [inline aqueduct/kubernetes/api-deployment-and-service.yaml]: role: backend

File: aqueduct/kubernetes/api-deployment-and-service.yaml (hunk 1 above)
Comment (MarcusSmith) [reply aqueduct/kubernetes/api-deployment-and-service.yaml]: 👍

File: aqueduct/kubernetes/api-deployment-and-service.yaml (hunk 2 above)
Comment (MarcusSmith) [reply aqueduct/kubernetes/api-deployment-and-service.yaml]: 👍

**Pull request merged**